test/it/uk/gov/hmrc/individualsifapistub/DetailsRepositorySpec.scala

```diff
@@ -60,10 +58,11 @@ class DetailsRepositorySpec extends RepositoryTestHelper with TestHelpers {
       val id = s"${ident.nino.getOrElse(ident.trn.get)}-$useCase"
 
       val detailsResponse = DetailsResponse(id, request.contactDetails, request.residences)
+      val returnVal       = DetailsResponseNoId(detailsResponse.contactDetails, detailsResponse.residences)
```

Comment:
Just create returnVal outright from request? Seems unneccisary to add the step of making a detailsResponse and then putting into the return val when the detailsResponse is not used elsewhere in the test.

Reply:
@parkeradam  - this is so we drop the id from the response

Reply:
@ajchilts Yeah but I mean it can still be constructed by simply doing DetasilsResponseNoId(request.contactDetails, request.residences) and completely remove the details response line. Just uneeded code for the same output and takes more to read through whats happening.

Reply:
Ah - its the in spec. Im going mad. :-| 

Too much going on. Will fix.